pkg/reconciler/taskrun/controller.go

```diff
@@ -55,7 +56,8 @@ func NewController(opts *pipeline.Options, clock clock.PassiveClock, tracerProvi
 		limitrangeInformer := limitrangeinformer.Get(ctx)
 		verificationpolicyInformer := verificationpolicyinformer.Get(ctx)
 		resolutionInformer := resolutioninformer.Get(ctx)
-		configStore := config.NewStore(logger.Named("config-store"), taskrunmetrics.MetricsOnStore(logger))
+		spireControllerAPI := spire.GetControllerAPIClient(ctx)
```

Comment:
nit: since this is assigned to SpireClient, rename to spireClient
```suggestion
		spireClient := spire.GetControllerAPIClient(ctx)
```